mkosi/__init__.py

```diff
@@ -4894,7 +4898,9 @@ def save_manifest(args: MkosiArgs, manifest: Manifest) -> None:
 
 
 def print_output_size(args: MkosiArgs) -> None:
-    if args.output_format in (OutputFormat.directory, OutputFormat.subvolume):
+    if not args.output.exists():
+        return
+    elif args.output_format in (OutputFormat.directory, OutputFormat.subvolume):
```

Comment:
This should just stay an `if`, we already shortcut execution with the `return`. Also needs a newline after the `return`

Reply:
Done